src/test/cucumber/flush.feature

```diff
@@ -37,3 +37,12 @@ Feature: Flush
     When I enter "sdk flush temp"
     Then no "res-1.2.0.zip" file is present in temporary storage
     And I see "1 archive(s) flushed"
+
+  Scenario: Clear out the cached archives and the temporary space
+    Given the archive "grails-1.3.9.zip" has been cached
+    And the file "res-1.2.0.zip" in temporary storage
+    When I enter "sdk flush all"
```

Comment:
I'm wondering if `all` isn't redundant. Following the way the rest of the CLI works, we try making things as lean as possible. How about a simple `sdk flush` being responsible for flushing _all_. wdyt?

Reply:
Hi Marc, I personally like your idea of a `sdk flush` being used to flush temp and archives.
There are just 2 questions coming into my mind:
1) Is it what a user would expect to happen when by simply typing `sdk flush ` ? If so, yes, let us do it that way.
2) What should the help page display ? 
`       flush             [archives|temp|broadcast|version]`
instead of 
`        flush             <broadcast|archives|temp>`

About your question:

> Should this duplicate line be here? And what does flushing temporary storage print to console?

The line isn't duplicate if both temp and archives having 'data' the new all `sdk flush all` would print 2 lines: 
```
1 archive(s) flushed, freeing 13M       /home/koko/.sdkman/archives.
1 archive(s) flushed, freeing 16K       /home/koko/.sdkman/tmp.
```
So the I tried picking this 2 lines with cucumber - might have been the wrong atempt.